reporters/cloud-run-scripts/CloudRunnerHub.js

```diff
@@ -140,4 +189,8 @@ async function _postJson(url, body) {
 const port = process.env.PORT || 8080;
 app.listen(port, () => {
   console.log("Listening on port", port);
+  // The cloud runner hub should have a configured URL to define the remote instance & a local node URL to boot.
+  if (!process.env.PROTOCOL_RUNNER_URL || !process.env.CUSTOM_NODE_URL) {
+    throw new Error("Bad environment! Specify a `PROTOCOL_RUNNER_URL` to point to the a cloud run instance");
+  }
```

Comment:
Does an error in this callback cause the process to crash on creation of the cloud run instance? Or does it fail silently? Or does it only fail when a request comes in?

Reply:
It fails when creating the cloud run instance & the log is shown in the GCP cloud run logs page.